tests/testthat/test-nice_fit.R

```diff
@@ -43,3 +43,8 @@ test_that("nice_fit labels", {
   )
 })
 
+
+test_that("nice_fit warns on labels", {
+  expect_warning(
+    nice_fit(fit, fit, model.labels = "Second Model"))
```

Comment:
can you also add an expect error test with a vector that is too large?

Reply:
Nice idea! Will implement this 💯

Reply:
Done by 9941267bab42082c1c4d0a8395ae4a2e31d36b65